scripts/covid19indiaORG/Covid19IndiaORG.py

```diff
@@ -0,0 +1,167 @@
+    for date, data in timeseries_data.items():
+        if 'total' in data:
+            total = data['total']
+            output[date] = total
+    return output
+
+
+def _get_state_table(iso_code: str, state_data: Dict[str, Dict[str, int]]):
+    """
+    Given Covid 19 India JSON data for a specific state,
+    return the data in the form of a DataFrame including
+    the state and the districts within the state.
+    @param state_data: the data for a specific state.
+    """
+    # Parse the timeseries for the given state.
+    state_timeseries: Dict[str, Dict[str, int]] = \
+         _parse_timeseries(state_data)
+
+    # Store the data in a DataFrame, and transpose it.
+    # Transposing puts the dates in the Y-axis.
+    state_table = pd.DataFrame(state_timeseries).T
+
+    # Get the WikiDataId from the dictionary.
+    state_table['wikidataId'] = STATES[iso_code]
+
+    # The district data is stored under 'districts'.
+    if 'districts' in state_data:
+        # Get the all district data for the state.
+        districts_data = state_data['districts']
+    else:
+        districts_data = {}
+        district_table = pd.DataFrame({})
+
+    # For every district in the state, parse the timeseries.
+    for district_name, data in districts_data.items():
+        district_timeseries = _parse_timeseries(data)
+        # Store the data in a DataFrame, and transpose it.
+        # Transposing puts the dates in the Y-axis.
+        district_table = pd.DataFrame(district_timeseries).T
+
+        # If there is not wikidataId for the given district name, skip it.
+        if not (iso_code in DISTRICTS and district_name in DISTRICTS[iso_code]):
+            continue
+
+        # Get the WikidataId from the dictionary.
+        district_table['wikidataId'] = DISTRICTS[iso_code][district_name]
+
+    # Store the DataFrame for this district under the state table.
+    state_table = pd.concat([state_table, district_table])
+
+    if 'confirmed' in state_table and 'recovered' in state_table:
+        # Calculate the active cases = confirmed cases - recovered cases.
+        state_table['active'] = state_table['confirmed'] - state_table['recovered']
+
+    # Get rid of any rows that don't have a wikidataId.
+    state_table.dropna(subset=['wikidataId'], inplace=True)
+
+    # Find the columns in common between the table and the Config.DATA_TO_KEEP.
+    columns_to_keep = DATA_TO_KEEP.intersection(set(state_table.columns))
+
+    # Get rid of any rows that don't have at least one of the following.
+    state_table.dropna(subset=list(columns_to_keep), thresh=1, inplace=True)
+
+    # Only keep the following columns, the rest are not part of the import.
+    state_table = state_table[list(columns_to_keep ) + ['wikidataId']]
+
+    # Rename the Y-axis to be "date".
+    state_table.index.name = "date"
+
+    return state_table
+
+def Covid19IndiaORG(state_to_source: Dict[str, str] = None, output: str = "output.csv"):
```

Comment:
What if we didn't have default values, and instead passed in STATE_APIS and "output.csv" from 'main'  ? I feel that is cleaner and less code.

Reply:
done.